Source/ARTDefault.m

```diff
@@ -34,6 +35,10 @@ + (NSString *)version {
     return ARTDefault_version;
 }
 
++ (NSString *)libVersion {
+    return [@[ARTDefault_platform, ARTDefault_version] componentsJoinedByString:@"-"];
```

Comment:
Thanks, but I don't think `ARTDefault_version` is what we are after, I believe that represents the API version, not the client library version. Perhaps introduce a new var and fix the old one for clarity.